scripts/install/autodl.sh

```diff
@@ -73,22 +73,22 @@ WorkingDirectory=/home/%i/
 WantedBy=multi-user.target
 ADC
 
-for u in "${users[@]}"; do
-  systemctl enable -q --now irssi@${u} 2>&1  | tee -a $log
-done
-echo_progress_done
+	for u in "${users[@]}"; do
+		systemctl enable -q --now irssi@${u} 2>&1 | tee -a $log
+	done
+	echo_progress_done
 }
 
 users=($(cut -d: -f1 < /etc/htpasswd))
 
 if [[ -n $1 ]]; then
-  users=($1)
-  _autoconf
-  exit 0
+	users=($1)
+	_autoconf
```

Comment:
On closer inspection, there's nothing to `enable` the service for new users (behaviour carried over from old script)

Reply:
Made a separate issue for this so we won't lose it. I'll mark this as resolved now so we can merge this